ports/esp32/boards/UM_TINYPICO/sdkconfig.board

```diff
@@ -3,3 +3,5 @@ CONFIG_ESPTOOLPY_FLASHFREQ_80M=y
 CONFIG_SPIRAM_SPEED_80M=y
 CONFIG_ESP32_REV_MIN_1=y
 CONFIG_LWIP_LOCAL_HOSTNAME="UMTinyPICO"
+
+CONFIG_COMPILER_OPTIMIZATION_SIZE=y
```

Comment:
Would be good to add a comment here, like "Enabling SPIRAM increases the size of the firmware, so use -Os to reduce it again".